docker-vertica-v2/Dockerfile

```diff
@@ -101,6 +101,7 @@ ENV PATH "$PATH:/opt/vertica/bin:/opt/vertica/sbin"
 ENV DEBIAN_FRONTEND noninteractive
 
 COPY ./packages/init.d.functions /etc/rc.d/init.d/functions
+COPY ./packages/cleanup-initial.sh /tmp/cleanup.sh
```

Comment:
Can we retain the name of the file that we copy into the image? I think that helps clarity, especially since we already have a cleanup.sh.